### PR TITLE
add arUSD from Acrechain

### PR DESCRIPTION
**Pull request by gophereth**

### CODE REVIEW

**Comment by gophereth** [comment]
@JeremyParish69 can you check the PR ?

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -2380,6 +2380,13 @@ const chainInfos = (
             high: 40000000000,
           },
         },
+        {
+          coinDenom: "arUSD",
+          coinMinimalDenom: "erc20/0x2Cbea61fdfDFA520Ee99700F104D5b75ADf50B0c",
+          coinDecimals: 18,
+          coinGeckoId: "pool:erc20/0x2Cbea61fdfDFA520Ee99700F104D5b75ADf50B0c",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:2387]: Please declare this id

Comment (gophereth) [reply packages/web/config/chain-infos.ts:2387]:
```
  {
    alternativeCoinId: "pool:erc20/0x2Cbea61fdfDFA520Ee99700F104D5b75ADf50B0c",
    poolId: "895",
    spotPriceSourceDenom: DenomHelper.ibcDenom(
      [{ portId: "transfer", channelId: "channel-490" }],
      "erc20/0x2Cbea61fdfDFA520Ee99700F104D5b75ADf50B0c"
    ),
    spotPriceDestDenom: DenomHelper.ibcDenom(
      [{ portId: "transfer", channelId: "channel-208" }],
      "uusdc"
    ),
    destCoinId: "usd-coin",
  },
```
Added this

**Review by JeremyParish69 — APPROVED** [review]
LGTM

**Pull request merged**